src/index.html

```diff
@@ -11,7 +11,194 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post" 
+      id="form"> 
+      <fieldset class="fieldSet">
+        <legend>Personal Information</legend>
+     
+      
+        <div>
+          <label for="sname">Surname:</label>
+          <input class="form_element"
+            type="text" 
+            name="surname"
+            id="sname"
+          > <br>
```

Comment:
```suggestion
          <input 
            class="form_element"
            type="text" 
            name="surname"
            id="sname"
          > <br>
```
Please fix all cases